lib/lambda_ethereum_consensus/execution/eth1_chain.ex

```diff
@@ -0,0 +1,128 @@
+defmodule LambdaEthereumConsensus.Execution.Eth1Chain do
```

Comment:
Maybe we could call it ExecutionChain (to get out of the legacy Eth1 nomenclature) and perform other duties like checking engine capabilities and making sure that we're on the proper chain (out of scope for this PR)

Reply:
Sure!